dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/AssignmentNode.java

```diff
@@ -26,6 +27,8 @@
  *
  * <p>Some desugarings create additional assignments to synthetic local variables. Such assignment
  * nodes are marked as synthetic to allow special handling in transfer functions.
+ *
+ * <p>StringConcatenateAssignmentNode is parsed as an Assignment node now.
```

Comment:
```suggestion
 * <p>String concatenation compound assignments are desugared to an assignment and a concatenation.
```

Reply:
Don't focus on the change, document what is. Don't use a class that doesn't exist anymore in the documentation.
What is the proper term for it?

Reply:
The examples and this text don't mention that other compound-assignments are also desugared. Might be better to also fix that problem now.

Reply:
fixed.